proto/palomachain/paloma/gravity/query.proto

```diff
@@ -101,10 +106,10 @@ message QueryDenomToERC20Response { string erc20 = 1; }
 // actual Ethereum block height significantly due to 1. Ethereum Finality and
 //  2. Consensus mirroring the state on Ethereum
 message QueryLastObservedEthBlockRequest {
-  // indicates whether to search for store data using the old Gravity v1 key
-  // "LastObservedEthereumBlockHeightKey" Note that queries before the Mercury
-  // upgrade at height 1282013 must set this to true
-  bool use_v1_key = 1;
+  reserved 1;
+  reserved "use_v1_key";
```

Comment:
This parameter seems to have been strictly focussed on the Gravity project with no impact on Paloma whatsoever.